src/commands/clone.ts

```diff
@@ -20,11 +20,12 @@ import {
 
 const padEnd = require('string.prototype.padend');
 const prompt = require('inquirer').prompt;
+const scriptIdStandardLength = 57;
```

Comment:
May be nice to create a `constants.ts` file, similar to `utils.ts` just for constants like this that may be shared across functions.